plugins/copy-to-clipboard/index.html

```diff
@@ -20,14 +20,165 @@
 
 <section>
 	<h1>How to use</h1>
-	<p>In addition to including the plugin file with your PrismJS build, ensure <a href="https://clipboardjs.com/">Clipboard.js</a> is loaded before the plugin.</p>
 
-	<p>The simplest way to include Clipboard.js is to use any of the
-		<a href="https://github.com/zenorocha/clipboard.js/wiki/CDN-Providers">recommended CDNs</a>. If you're using Browserify, Clipboard.js will be loaded automatically
+	<p>The plugin depends on <a href="https://clipboardjs.com/">clipboard.js</a> and the Prism <a href="https://prismjs.com/plugins/toolbar/">Toolbar</a> plugin. In addition to including the plugin file with your PrismJS build, ensure they are loaded before the plugin.</p>
+
+	<p>The simplest way to include clipboard.js is to use any of the
+		<a href="https://github.com/zenorocha/clipboard.js/wiki/CDN-Providers">recommended CDNs</a>. If you're using Browserify, clipboard.js will be loaded automatically
 		if it's included in your <code>package.json</code>.
-		If you don't load Clipboard.js yourself, the plugin will load it from a CDN for you.</p>
+		If you don't load clipboard.js yourself, the plugin will load it from a CDN for you.</p>
+</section>
+
+<section>
+	<h1>Settings</h1>
+
+	<p>By default, the plugin shows messages in English and sets a 5-second timeout after a click. You can use the following HTML5 data attributes to override the default settings:</p>
+
+	<ul>
+		<li><code class="token attr-name">data-prismjs-copy</code> — a copy-to-clipboard message;</li>
+		<li><code class="token attr-name">data-prismjs-copy-error</code> — a copying error message;</li>
+		<li><code class="token attr-name">data-prismjs-copy-success</code> — a message for a successful copying;</li>
+		<li><code class="token attr-name">data-prismjs-copy-timeout</code> — a timeout (in milliseconds) after copying. The value should be a non-negative integer.</li>
```

Comment:
People that don't know the message cycle probably won't understand the settings from those descriptions alone. How about this:

```suggestion
		<li><code class="token attr-name">data-prismjs-copy</code> — default message displayed by Copy to Clipboard;</li>
		<li><code class="token attr-name">data-prismjs-copy-error</code> — a message displayed after failing copying, prompting the user to press <code>Ctrl+C</code>;</li>
		<li><code class="token attr-name">data-prismjs-copy-success</code> — a message displayed after a successful copying;</li>
		<li><code class="token attr-name">data-prismjs-copy-timeout</code> — a timeout (in milliseconds) after copying. Once the timeout passed, the success or error message will revert back to the default message. The value should be a non-negative integer.</li>
```